package.json

```diff
@@ -1,12 +1,10 @@
 {
   "private": true,
-  "workspaces": [
-    "addon",
-    "test-app",
-    "docs-app"
-  ],
+  "scripts": {
+    "preinstall": "npx only-allow pnpm"
```

Comment:
pnpm-maintained package, suggested by https://dev.to/andreychernykh/yarn-npm-to-pnpm-migration-guide-2n04